packages/ui/v2/modules/booker/DatePicker.tsx

```diff
@@ -146,6 +172,10 @@ const Days = ({
           )}
         </div>
       ))}
+
+      {!props.isLoading && includedDates && includedDates?.length === 0 && (
```

Comment:
there is a very short flicker like 0.1ms on first load. maybe there is another prop we can check for here